src/aframe-uipack-utils.js

```diff
@@ -122,8 +138,7 @@ AFRAME_UIPACK.utils = {
 
         }
 
-
-        AFRAME_UIPACK.cursor_mode = desktop ? "desktop" : (mobile ? "gaze" : "laser");
-
+        mobile_touch_ui = mobile && mobile_touch_ui;
+        AFRAME_UIPACK.cursor_mode = (desktop || mobile_touch_ui) ? "desktop" : (mobile ? "gaze" : "laser");
```

Comment:
A lot of the work is done here, by using "desktop" mode you can use touch to use the UI with gaze.